consensus/processor.go

```diff
@@ -26,6 +26,12 @@ var (
 	// Timers
 	amTimer = metrics.NewTimerMs("consensus/apply_message", "Duration of message application in milliseconds", msgMethodKey)
 	pbTimer = metrics.NewTimerMs("consensus/process_block", "Duration of block processing in milliseconds")
+	ptTimer = metrics.NewTimerMs("consusnsu/process_tipset", "Duration of tipset processing in milliseconds")
```

Comment:
nit: spelling.  Though I'd like to consider `consusnsu` as an `flarp` alternative cc @laser

Reply:
"con-suu-snuuu"